lavalink/models.py

```diff
@@ -223,7 +228,11 @@ def add(self, requester: int, track: dict, extra: dict = None, index: int = None
                 track = self.queue.pop(0)
 
         self.current = track
-        await self.node._send(op='play', guildId=self.guild_id, track=track.track, startTime=start_time)
+        if end_time is -1:
```

Comment:
`==`

Reply:
I think modifying function parameter values to shave a few ns off processing times is probably pointless here, but I do agree that perhaps we shouldn't be sending stuff to Lavalink that doesn't need to be sent.